### PR TITLE
Add logLevel option

### LINKED ISSUE #10: Option to disable log messages
The font handling code contains some logs like

> Font 'xxx/xxx.ttf':0 found in 0.001ms.

Is it possible to disable them?

### PR DESCRIPTION
**Pull request by axelhzf**
This PR adds a `logLevel` option to handle the granularity of the log messages. 

Fix #10

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ svgtypes = "0.8.0"
 tiny-skia = "0.6.1"
 fontdb = "0.7.0"
 log = "0.4.11"
+env_logger = "0.8.4"
```
Comment (yisibl) [inline Cargo.toml:20]: Can we use 0.9.0? https://github.com/env-logger-rs/env_logger/releases/tag/v0.9.0

File: src/options.rs [hunk 2]
```diff
@@ -107,6 +124,7 @@ impl Default for JsOptions {
       fit_to: usvg::FitTo::Original,
       background: None,
       crop: JsCropOptions::default(),
+      log_level: log::LevelFilter::Debug
```
Comment (yisibl) [inline src/options.rs:127]: It would be better to enable only `Error` by default

Comment (axelhzf) [reply src/options.rs]: yep, that makes more sense

**Comment by yisibl** [comment]
LGTM. Thanks

**Pull request merged**